src/validations/request.validation.js

```diff
@@ -1,7 +1,14 @@
-import Joi from '@hapi/joi';
+import joiBase from '@hapi/joi';
+import joiDate from '@hapi/joi-date';
+import Sequelize from 'sequelize';
 import ResponseService from '../services/response.service';
 import TripService from '../services/trip.service';
 import RequestService from '../services/request.service';
+import LocationService from '../services/location.service';
+
+const { Op } = Sequelize;
```

Comment:
instead of declaring a variable here, you should destructure Op where you imported sequelize

Reply:
Ok did it